upgrade.js

```diff
@@ -0,0 +1,30 @@
+export const upgradeScripts = [
+	function v2_1(context, props) {
```

Comment:
Is this upgrade script needed? the name of it suggests it isnt relevant for your module?